<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>1</number>
       </property>
       <item>
        <widget class="QToolButton" name="btn_buat">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Buat Peta Bidang</string>
         </property>
         <property name="statusTip">
          <string/>
         </property>
         <property name="text">
          <string>Buat Peta</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../images/panel_icons/adddatabase.png</normaloff>../../images/panel_icons/adddatabase.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
         <property name="arrowType">
          <enum>Qt::NoArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btn_mulai">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Mulai</string>
         </property>
         <property name="statusTip">
          <string/>
         </property>
         <property name="text">
          <string>Mulai</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../images/panel_icons/start.png</normaloff>../../images/panel_icons/start.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btn_simpan">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Simpan PBT</string>
         </property>
         <property name="text">
          <string>Simpan</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../images/panel_icons/save.png</normaloff>../../images/panel_icons/save.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btn_link">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Masukkan Bidang ke Berkas</string>
         </property>
         <property name="text">
          <string>Link</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../images/panel_icons/linkdocument.png</normaloff>../../images/panel_icons/linkdocument.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btn_layout">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Layout</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../images/panel_icons/print.png</normaloff>../../images/panel_icons/print.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btn_tutup">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Tutup</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../images/panel_icons/close.png</normaloff>../../images/panel_icons/close.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btn_selesai">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Selesai</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../images/panel_icons/finish.png</normaloff>../../images/panel_icons/finish.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="layoutWidget_3">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Nomor PBT</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QgsFilterLineEdit" name="input_nomor_pbt">
               <property name="showSearchIcon">
                <bool>true</bool>
               </property>
               <property name="qgisRelation" stdset="0">
                <string notr="true"/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Tahun </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QgsFilterLineEdit" name="input_tahun">
               <property name="showSearchIcon">
                <bool>true</bool>
               </property>
               <property name="qgisRelation" stdset="0">
                <string notr="true"/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QLabel" name="kegiatanLabel_2">
             <property name="text">
              <string>Kegiatan</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QComboBox" name="combo_kegiatan">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QToolButton" name="btn_cari">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
        <property name="text">
         <string>Cari</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../images/panel_icons/search.png</normaloff>../../images/panel_icons/search.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="table_apbn">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btn_first">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_prev">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="input_page">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_next">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_last">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>&gt;&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFilterLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qgsfilterlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
